gh-93253: Add an explicit message for when there is not enough disk space to create new semaphores (#93254)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2022-05-26-11-44-51.gh-issue-93253.egSDbi.rst

> Added clear message
> 
> When there is not enough disk space to create new semaphores, a confusing error occurs.
> This fix adds a clearer message.

Reviewer: ```suggestion
Add an explicit message for when there is not enough disk space to create new semaphores.
```
